_data/simple-icons.json

```
@@ -13737,6 +13737,11 @@
             "hex": "D95E26",
             "source": "https://www.vyond.com"
         },
+        {
+            "title": "W3 Schools",
```

```suggestion
            "title": "W3Schools",
```

It's stylized most places as `W3Schools` (one word) - could you update this, and the title in the SVG?

---

Fixed